src/bitwise/core.cljs

```diff
@@ -147,7 +150,11 @@
                                  :disabled (= process-slots-available 0)} "<"]
                        [program-info program]])]]]
      [:div
-      [:h3 {:style {:margin-bottom 8}} "debug"]
+      [:h3 {:style {:margin-bottom 8}} "Developer Tools"]
+      [:div 
+        [action-button {:on-click #(swap! game-state assoc-in [:programs] (into #{} (keys program-catalog)))} "Grant All Programs"]
```

Comment:
As a point of interest, you can also use `assoc`, which takes a single keyword instead of a list of keywords. In practice, though, I like `assoc-in` even for single keywords because it's not much more overhead, and it's more flexible if the path changes.